app/src/main/java/org/fossasia/openevent/general/favorite/UserFavoriteApi.kt

```diff
@@ -0,0 +1,20 @@
+package org.fossasia.openevent.general.favorite
+
+import io.reactivex.Single
+import retrofit2.http.Body
+import retrofit2.http.DELETE
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+interface UserFavoriteApi {
+
+    @GET("user-favourite-events")
+    fun getFavorites(): Single<List<UserFavorite>>
+
+    @POST("user-favourite-events")
+    fun postFavorite(@Body favorite: UserFavorite): Single<UserFavorite>
```

Comment:
Can I ask you something because this is the part that confused me before so that I didn't make progress in this issue. If posting and getting favorite events like this then does it mean favorite events in my app is the same as favorite events in your app?

Reply:
nope it will be associated with your account only user id fetch from the JWT token and then user favorite are mapped with userid which JWT token passed as a header

Reply:
oh, thanks for the information

Reply:
@iamareebjamal should I create an issue on the server to add this  information to docs?

Reply:
What issue?